### PR TITLE
Challenge 1

### PR DESCRIPTION
**Pull request by RizalMadani**
Add new camperCell

### CODE REVIEW

File: challenges/challenge-1/challenge-1.css [hunk 1]
```diff
@@ -217,3 +217,19 @@ Saurav Kumar's styling
...
+  padding: 16px 24px;
+  background: #e8e8f7;
+  border: unset;
+  box-shadow: unset;
+  transition: 0.25s ease-in;
+}
+#rzl img{
+  margin: 16px 0;
+}
+#rzl:hover{
```
Comment (terrifricker) [inline challenges/challenge-1/challenge-1.css:231]: The animation looks great and was a good design choice to pair with the cleaner look of your cell.

**Review by terrifricker — APPROVED** [review]

**Comment by terrifricker** [comment]
Your second commit is not necessary. Were you intending to pull from `DavidMattherFraser:master` into your `challenge-1` branch?  I am still learning all the ins and outs of git, but I think this is what you do to remove it from this pull request:

Back in your local environment, on your challenge-1 branch, do a hard reset to the commit below your merge commit, commit 417368f.  Then push your challenge-1 branch to upstream master again.

Please ask about anything above that doesn't make sense, it will help me to understand better as well.

**Comment by terrifricker** [comment]
I went ahead and merged your pull request, so you don't need to do a hard reset anymore.

File: challenges/challenge-1/challenge-1.css (hunk 1 above)
Comment (RizalMadani) [reply challenges/challenge-1/challenge-1.css:231]: Thanks, I just like clean minimalist design

**Comment by RizalMadani** [comment]
Yeah, I was intended to update my branch with the upstream. But thanks if it was solved and my branch could be merged now 😁

**Pull request merged**